src/Conference/ConferenceDetails.test.js

```diff
@@ -41,30 +46,40 @@ afterEach(() => {
 
 describe("ConferenceDetails", () => {
   const mockData = {
-    data: {
-      id: 1,
-      name: "Festival of Marketing",
-      dateTime: "2019-11-12T12:34:11Z",
-      city: "London",
-      description:
-        "From Festivalofmarketing.com: The Festival of Marketing is a unique experience where ambitious marketers can discover, learn, celebrate and shape the future together. As the largest global event dedicated to brand marketers, the Festival reflects the very nature of ...",
-      topic: "Marketing"
-    }
+    id: 1,
+    name: "Festival of Marketing",
+    dateTime: "2019-11-12T12:34:11Z",
+    city: "London",
+    description:
+      "From Festivalofmarketing.com: The Festival of Marketing is a unique experience where ambitious marketers can discover, learn, celebrate and shape the future together. As the largest global event dedicated to brand marketers, the Festival reflects the very nature of ...",
+    topic: "Marketing"
   };
 
-  it("renders conference details", done => {
-    expect.assertions(1);
+  it("renders conference details", async () => {
+    expect.assertions(6);
 
-    axios.get.mockResolvedValue(mockData);
+    const history = createMemoryHistory();
+    history.push("/1");
 
-    useParams.mockReturnValue({ id: "1" });
+    const apiReturnValue = Promise.resolve(mockData);
+    getConferenceById.mockImplementation(() => apiReturnValue);
 
-    const component = mount(<ConferenceDetails />);
+    let wrapper;
+    await act(async () => {
+      wrapper = mount(
+        <Router history={history}>
+          <ConferenceDetails />
+        </Router>
+      );
 
-    setImmediate(() => {
-      component.update();
-      expect(component).toMatchSnapshot();
-      done();
+      await apiReturnValue;
     });
+    wrapper.update();
+    expect(wrapper.find("h3.name").length).toBe(1);
+    expect(wrapper.find(".topic").length).toBe(1);
+    expect(wrapper.find(".date").length).toBe(1);
+    expect(wrapper.find(".time").length).toBe(1);
+    expect(wrapper.find(".city").length).toBe(1);
+    expect(wrapper.find(".description").length).toBe(1);
```

Comment:
I Think this is a good change, but I still think as you are returning the mocked data you should still check that the correct data is being displayed within the elements, currently all you have checked is that the element exists not the content. If someone told you just to test this please feel free to ignore

Reply:
We have made the decision to start lightly testing components such as these, due to time constraints. If we test everything as much as we would expect the FE will fall further behind 😔

Reply:
okay makes sense 👍